Native.Renderers.Example.Forms/Native.Renderers.Example.Forms.Android/Renderers/AndroidBarcodeCameraRenderer.cs

```diff
@@ -32,20 +34,53 @@ namespace Native.Renderers.Example.Forms.Droid.Renderers
 {
     /*
        By extending 'ViewRenderer' we specify that we want our custom renderer to target 'BarcodeCameraView' and
-       override it with our native view 'ScanbotCameraView'
+       override it with our native view, which is a 'FrameLayout' in this case (see layout/barcode_camera_view.xml)
     */
-    public class AndroidBarcodeCameraRenderer : ViewRenderer<BarcodeCameraView, ScanbotCameraView>, ICameraOpenCallback
+    public class AndroidBarcodeCameraRenderer : ViewRenderer<BarcodeCameraView, FrameLayout>, ICameraOpenCallback
     {
         public BarcodeCameraView.BarcodeScannerResultHandler HandleScanResult;
```

Comment:
Do we need `public` or `protected` fields here?

Reply:
I am pretty sure we can make them protected. I'll change that asap